web/web/emails/send_emails.py

```diff
@@ -0,0 +1,38 @@
+import smtplib
+from dotenv import load_dotenv
+load_dotenv()
+
+import os
+from email.mime.text import MIMEText
+
+EMAIL_ADDRESS = os.getenv('EMAIL_ADDRESS')
+PASSWORD = os.getenv('PASSWORD')
+
+server = smtplib.SMTP('smtp.gmail.com:587')
+server.ehlo()
+server.starttls()
+
+
+def send_email(subject, message, bcc):
+    '''Simple SMTP set up with gmail'''
+
+    print("hello")
```

Comment:
There's a couple of stray print statements like this one. Let's get them all 🔪

Reply:
Oops - thank you! will remove immediately